src/shared/creds-util.c

```diff
@@ -667,13 +667,9 @@ int encrypt_credential_and_warn(
         p += tsz;
         assert(p <= output_size);
 
-        if (DEBUG_LOGGING) {
-                size_t base64_size;
-
-                base64_size = DIV_ROUND_UP(p * 4, 3); /* Include base64 size increase in debug output */
-
+        size_t base64_size = DIV_ROUND_UP(p * 4, 3); /* Include base64 size increase in debug output */
```

Comment:
Do we assume that the compiler will ellide the devision if debug logging is off?

Reply:
i would not assume so.. te DEBU_LOGGING is used precisely to avoid this calculation